db/db_tunables.h

```diff
@@ -551,9 +551,10 @@ REGISTER_TUNABLE("foreign_db_allow_cross_class", NULL, TUNABLE_BOOLEAN, &gbl_fdb
 REGISTER_TUNABLE("foreign_db_resolve_local", NULL, TUNABLE_BOOLEAN, &gbl_fdb_resolve_local,
                  READONLY | NOARG | READEARLY, NULL, NULL, NULL, NULL);
 REGISTER_TUNABLE("foreign_db_push_remote", NULL, TUNABLE_BOOLEAN, &gbl_fdb_push_remote, NOARG, NULL, NULL, NULL, NULL);
+REGISTER_TUNABLE("foreign_db_push_remote_write", NULL, TUNABLE_BOOLEAN, &gbl_fdb_push_remote_write, NOARG, NULL, NULL, NULL, NULL);
 REGISTER_TUNABLE("foreign_db_push_redirect",
                  "Redirect fdb query to run via client instead of on server. (Default: off)", TUNABLE_BOOLEAN,
-                 &gbl_fdb_push_redirect_foreign, NOARG, NULL, NULL, NULL, NULL);
+                 &gbl_fdb_push_redirect_foreign, NOARG, NULL, NULL, fdb_push_write_update, NULL);
```

Comment:
Is this supposed to be set on "foreign_db_push_remote" or "foreign_db_push_redirect"

Reply:
right, foreign_db_push_remote_write

Reply:
It looks like fdb_push_write_update is under the redirect tunable and not the write tunable